projects/core/src/site-context/services/base-site-initializer.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable, OnDestroy } from '@angular/core';
+import { Observable, Subscription } from 'rxjs';
+import { tap } from 'rxjs/operators';
+import { ConfigInitializerService } from '../../config/config-initializer/config-initializer.service';
+import { getContextParameterDefault } from '../config/context-config-utils';
+import { SiteContextConfig } from '../config/site-context-config';
+import { BaseSiteService } from '../facade/base-site.service';
+import { BASE_SITE_CONTEXT_ID } from '../providers/context-ids';
+
+@Injectable({ providedIn: 'root' })
+export class BaseSiteInitializer implements OnDestroy {
+  constructor(
+    protected baseSiteService: BaseSiteService,
+    protected configInit: ConfigInitializerService
+  ) {}
+
+  protected subscription: Subscription;
+
+  /**
+   * Initializes the value of the base site
+   */
+  initialize(): void {
+    this.subscription = this.setFallbackValue().subscribe();
```

Comment:
We should first wait until the `context` config is stable (so we know the available/valid values for the basesites).

Reply:
Solved